### PR TITLE
add gcp psc private code host support

### PR DESCRIPTION
**Pull request by michaellzc**

### CODE REVIEW

File: content/departments/cloud/technical-docs/private-code-hosts/index.md
Comment (filiphaftek) [inline content/departments/cloud/technical-docs/private-code-hosts/index.md:339]: I woudl add other steps here as well:
```sh
mi2 generate cdktf
# extends network stack with Private Service Connect Endpoint
mi2 instance tfc deploy -auto-approve
```

After deploy, ask customer to approve connection if `connection_preference = "ACCEPT_MANUAL"`.

**Review by filiphaftek — APPROVED** [review]
Great doc! added one suggestion.

File: content/departments/cloud/technical-docs/private-code-hosts/index.md
Comment (michaellzc) [reply content/departments/cloud/technical-docs/private-code-hosts/index.md:339]: > After deploy, ask customer to approve connection if connection_preference = "ACCEPT_MANUAL".

i don't think it's needed. the customer has to add our project_id to allowlist in service attachment, the connection will be approved automatically.

**Pull request merged**
